manager/integration/tests/test_basic.py

```diff
@@ -2819,7 +2838,18 @@ def test_expansion_with_scheduling_failure(
     expanded_size = str(400 * Mi)
     volume.expand(size=expanded_size)
     wait_for_volume_expansion(client, volume_name)
+    for i in range(RETRY_COUNTS):
+        claim = core_api.read_namespaced_persistent_volume_claim(
+            name=test_pvc_name, namespace='default')
+        if claim.spec.resources.requests['storage'] == expanded_size and \
+                claim.status.capacity['storage'] == "400":
+            break
+        time.sleep(RETRY_INTERVAL)
+    assert claim.spec.resources.requests['storage'] == expanded_size
+    assert claim.status.capacity['storage'] == "400Mi"
     volume = client.by_id_volume(volume_name)
+    # volume.detach(hostId="")
+    volume = wait_for_volume_detached(client, volume_name)
```

Comment:
It's redundant. `wait_for_volume_expansion` already verified that volume detachment.

Reply:
removed